### PR TITLE
Address Travis check failures

### PR DESCRIPTION
**Pull request by edgararuiz**

### CODE REVIEW

**Comment by hadley** [comment]
Thanks!

File: .Rbuildignore [hunk 1]
```diff
@@ -10,4 +10,5 @@
 ^cran-comments\.md$
 ^revdep$
 ^\.httr-oauth$
+^NEWS\.md$
```
[the remark this answers is not in the document]
Comment (edgararuiz) [reply .Rbuildignore]: Hi! thanks @etienne-s , I'm able to see it here: https://cran.r-project.org/web/packages/dbplyr/readme/README.html . Is it missing somewhere else?

[the remark this answers is not in the document]
Comment (hadley) [reply .Rbuildignore]: It definitely shouldn't be excluded from the build anymore. I'll fix it.

[the remark this answers is not in the document]
Comment (etienne-s) [reply .Rbuildignore]: @edgararuiz That's the README, not the NEWS! The news are missing here: https://cran.r-project.org/package=dbplyr

**Pull request merged**